spring-integration-core/src/test/java/org/springframework/integration/support/converter/ObjectStringMessageConverterTests.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2002-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.support.converter;
+
+
+import java.nio.charset.StandardCharsets;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.integration.support.MessageBuilder;
+import org.springframework.messaging.Message;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+/**
+ * @author Falk Hanisch
+ * @since 6.2
+ *
+ */
+class ObjectStringMessageConverterTests {
+	private static final ArbitraryPayload PAYLOAD = new ArbitraryPayload(1234, "ÄÖÜ");
```

Comment:
We prefer to wrap every class member into blank lines, even the last method (or inner class, record) before final `}` should have a blank line after its declaration.

This way the code is much readable, especially during review here on GH.